Bottom: cc346790d9fd691fdce657c3a9b4c33e94343c97
Top:    941649396eb9e3d935ba7ded567be7edaa265305
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 19:47:38 +0000

Refresh of remove-unused-hid-method-inval

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 8d1354f..2a92514 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -2840,7 +2840,7 @@ lesstif_invalidate_lr (int l, int r, int t, int b, int last)
 void
 lesstif_invalidate_all (void)
 {
-  lesstif_invalidate_lr (0, 0, PCB->MaxWidth, PCB->MaxHeight, 1);
+  lesstif_invalidate_lr (0, PCB->MaxWidth, 0, PCB->MaxHeight, 1);
 }
 
 static int
